FsAutoComplete/CommandInput.fs

```diff
@@ -138,10 +224,11 @@ module CommandInput =
     | null -> Quit
     | input ->
       let reader = Parsing.createForwardStringReader input 0
-      let cmds = compilerlocation <|> helptext <|> declarations <|> lint <|> parse <|> project <|> completionTipOrDecl <|> quit <|> colorizations <|> error 
-      let cmd = reader |> Parsing.getFirst cmds
```

Comment:
Could we just keep this as it was please? The Error case is already handled by the final option in the line above. Obviously we need to include the parsers for your new commands though.